pack-blob-ext.ps1

```diff
@@ -0,0 +1,11 @@
+# FOR TESTING ONLY, DELETE BEFORE MERGE
```

Comment:
Note to remove before merge

Reply:
To simplify things for reviewers and to ensure test stuff like this isn't checked in, it's best if you just don't include it in the PR at all.

Reply:
Done